wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -1,3 +1,6 @@
+Skipping builds for arm32 (toolchain is marked optional)
+Skipping builds for arm64 (toolchain is marked optional)
+Skipping builds for roboRio (toolchain is marked optional)
```

Comment:
```suggestion
```

Reply:
VS-code spotless formatter shat itself, and I didn't realize it before pushing. Fixed in a force push